sqlair.go

```diff
@@ -140,6 +140,35 @@ func (q *Query) Decode(outputArgs ...any) (ok bool) {
 	return true
 }
 
+// All iterates over the query and decodes all the rows.
+// It fabricates all structs as needed.
+// It returns a list of rows, each row is a list of pointers to structs.
+// The structs appear in the syntactic order they appear in the query.
```

Comment:
I think the word 'syntactic' might be superfluous here

Reply:
Changed.